PokeAlarm/Alarms/Discord/DiscordAlarm.py

```diff
@@ -80,6 +80,15 @@ class DiscordAlarm(Alarm):
             "title": "The weather has changed!",
             "url": "<gmaps>",
             "body": "The weather around <lat>,<lng> has changed to <weather>!"
+        },
+        'quest': {
```

Comment:
We should stick with the plural version -- `quests` -- since everything else uses the plural version